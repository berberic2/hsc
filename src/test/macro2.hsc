<$macro hinz title:string>
<page title="gurk">
<(title)>
</page>
</$macro>

<$macro page /close title:string>
<html>
<head></head>
<body><$content></body>
</html>
</$macro>

<hinz title=mist>

