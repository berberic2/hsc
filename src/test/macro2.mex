test/macro2.hsc (16,1): Warning 15: required tag <TITLE> missing
