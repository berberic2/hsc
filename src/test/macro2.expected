<html>
<head></head>
<body>
mist
</body>
</html>

